modules/serverless-domain-mapping-custom-tls-cert.adoc

```diff
@@ -26,6 +16,20 @@ You can add an existing TLS certificate with a `DomainMapping` custom resource (
 ----
 $ oc create secret tls <tls_secret_name> --cert=<path_to_certificate_file> --key=<path_to_key_file>
 ----
++
+[IMPORTANT]
+====
+If you are using {SMProductName} as the ingress for your {ServerlessProductName} installation, or you are using a third party secret provider such as cert-manager, label this secret with the following `<key>:<value>` pair:
```

Comment:
[Side note cert manager is tp on OCP](https://docs.openshift.com/container-platform/4.10/security/cert_manager_operator/cert-manager-operator-install.html).

Reply:
I think this is a question for QE, not docs.

Reply:
@maschmid any thoughts or comments on this?

Reply:
So, at least for current (redhat) version of cert-manager, the cert-manager would not delete the label added manually to the secret, unless it would be explicitly set in the `secretTemplate`. The proper way of adding the label to the secret using the cert-manager though, is to use the "secret template" mechanism, which we do suggest below in the next paragraph, so, for me the wording is fine. 

If we'd wanted to be on the safer side for any hypothetical secret provider, we could say something like "make sure to configure the secret provider to add the following label to the secret"  instead of just the "label this secret with the following" to emphasize that if the secret is managed by X, you should configure X to add the label automatically, instead of adding the label manually to the secret.  (of course, we cannot know if this hypothetical third-party tool do have such a feature)